provisioning/baremetal_pod.go

```diff
@@ -46,6 +46,7 @@ const (
 	mariadbPwdEnvVar           = "MARIADB_PASSWORD"    // #nosec
 	cboOwnedAnnotation         = "baremetal.openshift.io/owned"
 	cboLabelName               = "baremetal.openshift.io/cluster-baremetal-operator"
+	apiLabelKey                = "api"
```

Comment:
I'm normally a fan of named constants, but in this case it's only used in one place and is for handling legacy support (i.e. not something we ever want to use again). On balance, I think this serves only to make it more difficult to tell what is going on when reading the code.

Reply:
Honestly, it is hard to predict which way a reviewer would go. I can remove this constant.